RFS/src/test/java/com/rfs/integration/EndToEndTest.java

```diff
@@ -0,0 +1,65 @@
+package com.rfs.integration;
+
+import org.junit.jupiter.api.Test;
+import com.rfs.framework.SimpleRestoreFromSnapshot;
+
+/**
+ * Tests focused on setting up whole source clusters, performing a migration, and validation on the target cluster
+ */
+public class EndToEndTest {
+
+    public Object sourceCluster;
+    public Object targetCluster;
+    public SimpleRestoreFromSnapshot simpleRfsInstance;
+
+    @Test
+    public void ES_v6_8_to_OS_v2_X_Migration() throws Exception {
+        // Setup
+        // PSEUDO: Create a source cluster running ES 6.8
+        // PSEUDO: Update global cluster state
+        //    - Change a 6.8 relevant setting
```

Comment:
Maybe I should pull this section until we have something more concrete defined, are there any cluster settings on the source cluster that should be taken into consideration when configuring the target cluster?

Reply:
That's an excellent question.  So far we've assumed that settings our outside the domain of the migration, but that's just an assumption and may change with time.  I'd say take it out for now; I'm confident it will pop back up soon if it's something we actually need to tackle.